x-pack/platform/plugins/shared/streams_app/ui_tests/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Given that `ui_tests` is neither a plugin or a package, perhaps `tsconfig.json` here is not necessary. I see that `observability_onboarding` seems to have [one](https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/observability_onboarding/ui_tests/tsconfig.json), but for example other plugins do not. I'm not too sure on this item, however.